judoscale-ruby/lib/judoscale/config.rb

```diff
@@ -66,28 +93,36 @@ def self.expose_adapter_config(config_instance)
       end
     end
 
-    attr_accessor :api_base_url, :report_interval_seconds, :max_request_size_bytes, :logger, :log_tag
-    attr_reader :dyno, :log_level
+    attr_accessor :api_base_url, :report_interval_seconds,
+      :max_request_size_bytes, :logger, :log_tag, :current_runtime_container
+    attr_reader :log_level
 
     def initialize
       reset
     end
 
     def reset
-      # Allow the API URL to be configured - needed for testing.
       @api_base_url = ENV["JUDOSCALE_URL"] || ENV["RAILS_AUTOSCALE_URL"]
       @log_tag = "Judoscale"
-      self.dyno = ENV["DYNO"]
       @max_request_size_bytes = 100_000 # ignore request payloads over 100k since they skew the queue times
       @report_interval_seconds = 10
+
       self.log_level = ENV["JUDOSCALE_LOG_LEVEL"] || ENV["RAILS_AUTOSCALE_LOG_LEVEL"]
       @logger = ::Logger.new($stdout)
 
       self.class.adapter_configs.each(&:reset)
-    end
 
-    def dyno=(dyno_string)
-      @dyno = Dyno.new(dyno_string)
+      if ENV["RENDER"] == "true"
+        instance = ENV["RENDER_INSTANCE_ID"].delete_prefix(ENV["RENDER_SERVICE_ID"]).delete_prefix("-")
+        @current_runtime_container = RuntimeContainer.new :render, ENV["RENDER_SERVICE_ID"], instance, ENV["RENDER_SERVICE_TYPE"]
+        @api_base_url ||= "https://adapter.judoscale.com/api/#{ENV["RENDER_SERVICE_ID"]}"
+      elsif ENV["HEROKU"] == "true"
```

Comment:
```suggestion
      elsif ENV["DYNO"].present?
```

I want to consider checking `DYNO` instead of `HEROKU` here. I understand that `HEROKU` feels a bit more explicit, but I think it adds an extra dependency where we don't need one—it's only more piece of information that needs to be kept in our brains, and it's one more env var to specify from the command line for testing.

Functionally, `DYNO` is the only env var we really *need* for Heroku, so I'd rather not add another env var dependency if we don't have to.